data/sheragi/archaeology missions.txt

```diff
@@ -301,7 +301,7 @@ mission "Deep Archaeology 5"
 ship "Timer Ship"
 	attributes
 		"heat dissipation" 1
-		"hull" 3
+		"hull" 5
```

Comment:
Why?

Reply:
Like I said to make it a bit longer but it's not needed, and I'm not particularly attached to it
Zitchas asked for this and I dont know where we stand about that